models/workspacemanager/mantid_workspace_provider.py

```diff
@@ -42,3 +42,9 @@ def get_workspace_handle(self, workspace_name):
         if isinstance(workspace_name, Workspace):
             return workspace_name
         return AnalysisDataService[workspace_name]
+
+    def get_workspace_name(self, workspace):
```

Comment:
Right... I didn't because I saw that `get_workspace_handle` didn't have a declaration. I'll just add both these methods to `models/workspacemanager/workspace_provider.py`